### DISCUSSION TITLE
[Feature]: Full audit log -> log every action in coolify

### DISCUSSION THREAD

**Discussion by peaklabs-dev** [coollabsio/coolify#3138] [Feature Requests]
### Minimal Reproduction (if possible, example repository)

Audit log everything literally every movement, every setting change everything each user does in coolify and display it

### Version

-

**Comment by Extremys** [comment]
Hello, it would be great to have the possiblity to put in place audit trail and make it queryable :)

**Comment by ByteAfterlife** [comment]
I think this would be useful for viewing what happened and when and by who

**Comment by HammyHavoc** [comment]
NetBird is a good example of this, IMO.
